webapp/src/templates/partials/reports_content.html

```diff
@@ -68,7 +68,16 @@
           </li>
           <li ng-if="selection.formatted.form && selection.formatted.content_type === 'xml'" ng-repeat="field in selection.displayFields" class="indent-{{field.depth}}">
             <label translate>{{field.label}}</label>
-            <p ng-hide="field.imagePath">{{field.value}}</p>
+            <p ng-hide="field.imagePath">
+              <span ng-if="labelIsIDorName(field.label)">
+                  <a class="name" ng-if="summary.subject._id || summary.fields.patient_uuid" ng-href="#/contacts/{{summary.subject._id || summary.fields.patient_uuid}}"
+                    ng-bind="field.value">
+                  </a>
```

Comment:
It's better to use `ui-sref` rather than `ng-href` because it generates the URL from a route. Try replacing this with...


```html
<a class="name" ng-if="summary.subject._id || summary.fields.patient_uuid" ui-sref="contacts.detail({ id: summary.subject._id || summary.fields.patient_uuid })">{{ field.value }}</a>
```